_data/nav.yml

```diff
@@ -61,6 +61,7 @@ How to use USWDS:
   - href: /documentation/video-tutorials/
   - href: /documentation/implementations/
   - href: /documentation/showcase/
+  - href: /documentation/sample-contract/
```

Comment:
@allie-shaw, I've added the page to the sidenav in `How to use USWDS`

![image](https://user-images.githubusercontent.com/3385219/91599160-7c824880-e92b-11ea-9998-ca9338cd7b86.png)